src/RequestsLibrary/SessionKeywords.py

```diff
@@ -555,6 +555,9 @@ def update_session(self, alias, headers=None, cookies=None):
         session = self._cache.switch(alias)
         session.headers = merge_setting(headers, session.headers)
         session.cookies = merge_cookies(session.cookies, cookies)
+    
+    def save_last_response(self, response):
+        self.builtin.set_global_variable('${LAST_RESPONSE}', response)
```

Comment:
this would be really global? maybe is safer suite global. Not sure whether there are any best practice in RF keywords development, any other keyword that works like this which scope use?

Reply:
I believe global would be more appropriate than suite scope as global would be accessible from all the levels of execution as you can refer here: https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#variable-priorities-and-scopes